spec/js/post_office_search_spec.ts

```diff
@@ -60,52 +59,84 @@ describe('Post Office Search', () => {
     }
   });
 
-  const getField = function () {
-    return getByLabelText(
-      document.body,
-      'in_person_proofing.body.location.po_search.address_search_label',
-    );
+  const getField = function (labelName: string) {
+    return getByLabelText(document.body, `in_person_proofing.body.location.po_search.${labelName}`);
   };
 
-  const getButton = function () {
+  const getSearchButton = function () {
     return getByRole(container, 'button', {
       name: 'in_person_proofing.body.location.po_search.search_button',
     });
   };
 
-  it('renders a post office search component', () => {
-    const field = getField();
-    expect(field).toBeEnabled();
-    expect(field).toBeVisible();
+  const testData = {
```

Comment:
I don't want this data to change so used const.  If we need more data, we can create another const and/or nest it in different blocks. I preferred having one object with keys rather than having 4 different variables for each key. I went with lowerCamelCase to be consistent with the file.